docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,109 @@
+** If an existing indicator match rule was configured to use threat indicator indices generated from {filebeat} version 7.x, updating the default threat indicator path to `threat.indicator` after you upgrade to {stack} version 8.0.0 and {agent} or {filebeat} version 8.0.0 configures the rule to use threat indicator indices generated by those later versions.
+** You must create separate rules to query threat intelligence indices created by {filebeat} version 7.x and version 8.0.0 because each version requires a different default threat indicator path value.
+
+[discrete]
+[[breaking-changes-8.0.0]]
+==== Breaking Changes
+* Removes the trusted application API. The trusted application interface retains current functionality, but now uses the exception list API ({pull}120134[#120134]).
+* Removes the list endpoint metadata API ({pull}119401[#119401]).
+* Lets you grant privileges for cases separately from {elastic-sec} privileges ({pull}113573[#113573], {pull}112980[#112980]). As a result of this change, you must update case privileges for existing roles _before_ upgrading to {stack} 8.0.0. Follow these steps:
+. Open the main menu and click *Management -> Stack Management -> Stack -> Upgrade Assistant*.
+. From the Upgrade Assistant page, review the Kibana deprecation warnings. A message prompts you to update role privileges because of changes to the {elastic-sec} Cases feature.
+. Click the message to open it, then click *Quick resolve*.
+. Refresh the page to verify the deprecation was resolved, then return to the guided steps on the Upgrade Assistant page.
+
+[discrete]
+[[new-features-8.0.0]]
+==== Features
+* Shows all historical alerts for a given rule on the rule details page, including those associated with previous versions of the rule ({pull}120053[#120053]).
+* Enhances the UI and functionality for the Rules and Rule Monitoring tables and enables actions on the Rule Monitoring table ({pull}119644[#119644]).
+* The Threat Intelligence view supports {agent}, {filebeat}, and custom integrations ({pull}116175[#116175]).
+* Allows exception lists to be exported and imported with detection rules ({pull}115144[#115144], {pull}118816[#118816]).
+
+[discrete]
+[[bug-fixes-8.0.0]]
+==== Bug fixes and enhancements
+* Enhances the UI for the Exceptions table; improves how dates are displayed in the Rules and Exceptions tables ({pull}117643[#117643], {pull}118940[#118940]).
+* Updates the mappings of the rule registry to ECS version 8.0.0 so that detection rules can process ECS version 8.0.0 data ({pull}123012[#123012]).
+* Allows you to create runtime fields from, and add them to the Alerts and Timeline tables ({pull}117627[#117627], {pull}114806[#114806]).
```

Comment:
Or:
```suggestion
* Uses Data Views to query instead of index pattern string, allowing for the addition of runtime fields in Timeline and Alert tables ({pull}114806[#114806], {pull}117601[#117601]).
* Create a new runtime field in Timeline and Alert tables from the field browser "Create field" button ({pull}117627[#117627]).
```

Reply:
Changing to "Allows you to create and add runtime fields from the Alert and Timeline tables" as discussed with @stephmilovic in Slack.